core/src/main/scala/kafka/server/AbstractFetcherManager.scala

```diff
@@ -37,11 +37,11 @@ abstract class AbstractFetcherManager[T <: AbstractFetcherThread](val name: Stri
   val failedPartitions = new FailedPartitions
   this.logIdent = "[" + name + "] "
 
-  newGauge(
+  val maxLag = newGauge(
```

Comment:
Just wondering if there was a good reason to turn these into vals?

Reply:
Not really, just thought of making it consistent for all metrics. Reverted it.